articles/libraries/machine-learning/basic-classification.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Quantum machine learning library
+description: Learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the Microsoft QDK.
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.basics
+---
+
+# Basic classification: Classify data with the QDK
+
+In this Quickstart, you will learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK. 
+
+In this guide we will use the half-moon dataset, using a classifier structure defined in Q#.
+
+## Execute the classifier from the host program
```

Comment:
You'll want to include a pre-requisites section here: installation of QDK, set up project etc